git-pr-merge

```diff
@@ -177,20 +185,39 @@ common::setup() {
     return 1
   fi
 
+  # If called with a PR number, check out that PR using `gh`. This is used
```

Comment:
Hold on, when did we switch from hub to gh?

Reply:
It looks like in 759e1b54402519aa190f68ef689a5af4132901e0 on Wed 10th November 2021.

Reply:
💰